.github/workflows/isotovideo-action.yml

```diff
@@ -9,11 +9,9 @@ jobs:
       image: "registry.opensuse.org/devel/openqa/containers/isotovideo:qemu-x86-jq"
     steps:
       - uses: actions/checkout@v2
-      - name: install jq
-        run: zypper -n in jq
```

Comment:
That's a good change but should be in a separate commit (does not have to be separate PR, just separate commit)